src/app/ReadClient.cpp

```diff
@@ -1271,6 +1272,15 @@ CHIP_ERROR ReadClient::DefaultResubscribePolicy(CHIP_ERROR aTerminationCause)
         return CHIP_ERROR_LIT_SUBSCRIBE_INACTIVE_TIMEOUT;
     }
 
+    if (aTerminationCause.IsIMStatus() && ClusterStatusCode(aTerminationCause).GetStatus() == Status::InvalidAction)
+    {
+        // InvalidAction in most cases means some Interaction Model mismatch with the target device
+        // That's not something that could be fixed by retry
```

Comment:
InvalidAction for a subscription in practice means either "you tried to subscribe to things that don't exist" or "you don't have the right ACLs".

First of all, the documentation should say that.

Second, for that second case, it may be the case that retrying until something fixes the ACLs is in fact the right behavior.

The PR description says "while this happens no other commands could be sent to device", but that sounds _very_ odd.  Why can't you send anything else while resubscribes are happening, especially if they go into backoff?

And given the backoff behavior, it's very much not clear to me that we want to make this change: the client will just back off, so in practice there should be minimal impact here.

Reply:
> The PR description says "while this happens no other commands could be sent to device", but that sounds very odd. Why can't you send anything else while resubscribes are happening, especially if they go into backoff?

I'll take a look more thoroughly on our code. Thanks. Closing for now